Streams/task3.js

```diff
@@ -0,0 +1,32 @@
+const { Readable, Writable } = require('stream');
+
+const fastReadableStream = new Readable({
+  read() {
+    for (let i = 0; i < 10; i++) {
+      const data = `Chunk ${i}\n`;
+      console.log(`Pushing: ${data}`);
+      if (!this.push(data)) {
+        console.log('Pause');
+        return;
+      }
+    }
+    this.push(null);
+  }
+});
+
+const slowWritableStream = new Writable({
+  write(chunk, _, cb) {
+    console.log(chunk.toString());
+    setTimeout(cb, 1000);
+  }
+});
+
+slowWritableStream.on('drain', () => {
+  console.log('slowWritableStream');
+});
+
+fastReadableStream.pipe(slowWritableStream);
```

Comment:
pipe internally handles back pressure, in order to do it manually you should use .write method and check the value returned by it.